### ISSUE TITLE
Absent (minor) islands

### ISSUE THREAD

**Issue by tauanajc** [nvkelso/natural-earth-vector#321]
Many small islands cannot be found even in the minor_islands map. Example, islands in the Faafu Atoll, Maldives: Magoodhoo, Dharanboodhoo, Nilandhoo, Bileiydhoo. These are all <2 km2 in area, none show in the minor_islands map, two of then show in the coastline map, but not in their real shape.

**Comment by nvkelso** [comment]
Can you tell me more about your goal?

Generally islands that size are at limits of natural earth’s intended precision.

**Comment by tauanajc** [comment]
Hi, Nathaniel,
Thanks for the super fast reply to the issue.

I am trying to plot the collecting sites of a field trip to the Maldives. We were collecting around the island of Magoohdoo in the Faafu Atoll. I’ve used natural earth maps to create maps with the same goal before, but this is the first with such small geographic scale. I tried many of NE maps including the minor islands, and Magoodhoo and the close by islands don’t show at all, so I thought I was plotting something wrong. But with the coastline I saw a couple of them, both with the same shape. I guess most are not represented or are with not very good resolution. I understand it might not be the goal of NE, I just got excited when I saw the small islands map, seemed like it would be perfect for this.

I want to get the outline to the islands and hopefully, but not absolutely necessary, the bathymetry around, and then plot my sites. My next attempt would be google maps in R, since the satellite view shows the islands very well, but haven’t tried yet.

Thanks for hearing me out, regardless of whether there will be a solution.
Best,
Tauana

__

Tauana Junqueira Cunha
Postdoctoral Fellow
Smithsonian Tropical Research Institute
http://tauanacunha.com
